lib/manageiq/providers/ibm_cloud/cloud_tools.rb

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
 
-require 'logging'
```

Comment:
This wasn't used here.  It was in the other file.  🔥 🗑️ 🚽

Reply:
There is `CloudTool` _and_ `CloudTools` :confused:

Reply:
🤣